package.json

```diff
@@ -68,7 +67,7 @@
     "build": "yarn compile && rimraf dist && tsc && npm pack && node rename-pack.js",
     "format": "prettier --write .",
     "format:check": "prettier --check .",
-    "prepare": "husky install"
```

Comment:
not sure that removing 'install' is correct - I remember I had to add it to actually work - uninstall all packages, run yarn, commit some changes and see will it format all files (both .ts and .sol)